flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/runtime/stream/sql/AsyncLookupJoinITCase.scala

```diff
@@ -99,7 +93,6 @@ class AsyncLookupJoinITCase(backend: StateBackendMode)
       "1,12,Julian",
       "3,15,Fabian")
     assertEquals(expected.sorted, sink.getAppendResults.sorted)
-    assertEquals(0, userTableSourceWith2Keys.getResourceCounter)
```

Comment:
can't get `userTableSourceWith2Keys.getResourceCounter` using `tEnv.connect` ?

Reply:
We can't fetch the actual table source `userTableSourceWith2Keys` if we use tEnv.connect or DDL. Thus we can't verify the counter in that table source instance.